x-pack/legacy/plugins/apm/server/routes/settings/agent_configuration.ts

```diff
@@ -31,6 +31,9 @@ export const agentConfigurationRoute = createRoute(core => ({
 export const deleteAgentConfigurationRoute = createRoute(() => ({
```

Comment:
There are a few routes in here without any `access:` tags:
```
path: '/api/apm/settings/agent-configuration',
```

```
method: 'GET',
path: '/api/apm/settings/agent-configuration/services',
```

```
path: '/api/apm/settings/agent-configuration/environments',
```

```
path: '/api/apm/settings/agent-configuration/agent_name',
```

```
method: 'POST',
path: '/api/apm/settings/agent-configuration/search',
```

Are there default being supplied somewhere?